processing/src/test/java/org/apache/druid/segment/virtual/ExpressionVectorSelectorsTest.java

```diff
@@ -96,25 +96,31 @@
   private static QueryableIndex INDEX;
   private static Closer CLOSER;
 
+  private static final String CPU_ARCH = System.getProperty("os.arch");
+
   @BeforeClass
   public static void setupClass()
   {
     CLOSER = Closer.create();
 
-    final GeneratorSchemaInfo schemaInfo = GeneratorBasicSchemas.SCHEMA_MAP.get("expression-testbench");
+    // Do not run the tests on ARM64.
+    // SegmentGenerator#generate() fails with OutOfMemoryError on TravisCI ARM64
```

Comment:
maybe it would be trying with a smaller segment size (`ROWS_PER_SEGMENT`) instead of skipping the test entirely?

Reply:
I am not 100% certain but I think there is a memory leak in an earlier test(s).
ExpressionVectorSelectorsTest does not fail on my Ubuntu 20.04 ARM64 VM. I have tried even with much lower max memory - it passes with -Xmx384m. The test suite at TravisCI runs with -Xmx3000m.
Locally with -Xmx256m it fails with OOME Too much garbage collections.

Reply:
Tried with 2 smaller ROWS_PER_SEGMENT with https://github.com/apache/druid/pull/11109/commits/48ab08c0c8ea1c76f2d5a706ada563ba897f9720. Let's see!

Reply:
@clintropolis With this change the unit test passed: https://travis-ci.com/github/apache/druid/builds/223430180

Some other non-related TravisCI jobs failed for reasons not related to this PR, but the new ARM64 ones passed successfully.

Reply:
:+1:, in that case please remove this commented section and the no longer used `CPU_ARCH`.

Reply:
Done! 
Also rebased to latest `master`.